libzeth/tests/circuits/prfs_test.cpp

```diff
@@ -117,7 +119,7 @@ TEST(TestPRFs, TestPRFAddrApkGadget)
             0, 0, 0, 0, 0, 0, 0, 0, // 00
             1, 1, 1, 1, 1, 1, 1, 1  // FF
         },
-        ZERO);
+        "sk");
```

Comment:
Why not stick to the variable name here for the annotation? (same question for the lines below, e.g. why "expected0"/"expected1" and not "rho_expected0" etc.?)

Reply:
changed the annotation string

Reply:
The same comment applied for all the file where I can see that annotations differ from the variable names in many places.